queries/odin/rainbow-delimiters.scm

```diff
@@ -0,0 +1,91 @@
+(parameters
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(tuple_type
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(parenthesized_expression
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(polymorphic_parameters
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(polymorphic_type
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(attribute
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
```

Comment:
I don't know Odin, can `attribute` ever be nested within other delimiters or can it contain nested delimiters, or is it always a top-level element? If it's the latter I don't know if it should really be included in the query. What's your opinion?

Reply:
Even if it does not contain other delimiters, but can be contained inside other delimiters that's reason enough to include it.

> there are some examples where it contains other delimiters

I think that's good enough of a reason even if it is rare.